airbyte-config/init/src/main/resources/seed/source_specs.yaml

```
@@ -14481,7 +14481,7 @@
     supportsNormalization: false
     supportsDBT: false
     supported_destination_sync_modes: []
-- dockerImage: "airbyte/source-stripe:3.0.0"
+- dockerImage: "airbyte/source-stripe:3.0.1"
```

```suggestion
- dockerImage: "airbyte/source-stripe:3.0.0"
```

This will also get bumped automatically.